CHANGELOG.md

```diff
@@ -5,6 +5,51 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2025-06-10
```

Comment:
Would you prefer delaying the 0.3.0 release so that the `hayagriva` feature I am proposing in https://github.com/AliAzlanDev/biblib/issues/3 can be included in the same 0.3.0 release?

Reply:
You may also wish to merge https://github.com/AliAzlanDev/biblib/pull/5 before releasing v0.3.0